common/app/routes/Challenges/redux/execute-challenge-epic.js

```diff
@@ -89,7 +93,10 @@ export function executeChallengeEpic(actions, { getState }, { document }) {
                 // make sure that the test message is a non empty string
                 .filter(_.overEvery(_.isString, Boolean))
                 .map(updateOutput)
-                .concat(Observable.of(updateTests(tests)));
+                .concat(
+                  Observable.of(updateTests(tests)),
+                  Observable.of(updateSuccessMessage(randomCompliment()))
```

Comment:
If would be better if we could set the success message in the same way we handle translation messages in `common/app/routes/Challenges/Show.jsx`. That way, we are only calling it once per challenge.

This will lead to fewer state updates -> fewer re-renders -> better perf

```
if the nextDashedName !== currentDashedName, update the sucess message
```